pkg/util/mysqlsh/mysqlsh.go

```diff
@@ -74,31 +80,28 @@ func (r *runner) IsClustered(ctx context.Context) bool {
 	return err == nil
 }
 
-func (r *runner) CreateCluster(ctx context.Context, multiMaster bool) (*innodb.ClusterStatus, error) {
+func (r *runner) CreateCluster(ctx context.Context, multiMaster bool) error {
 	var python string
 	if multiMaster {
 		python = fmt.Sprintf("dba.create_cluster('%s', {'force':True,'multiMaster':True})", innodb.DefaultClusterName)
 	} else {
 		python = fmt.Sprintf("dba.create_cluster('%s')", innodb.DefaultClusterName)
 	}
 	_, err := r.run(ctx, python)
-	if err != nil {
-		return nil, fmt.Errorf("creating cluster: %v", err)
```

Comment:
So the reason we wrap the errors is sometimes they are not helpful and it's hard to tell which script run caused the issue. I'd keep them unless you are certain that they will always return useful errors.

Reply:
Generally agree (do love `errors.Wrap`). In this case though the underlying error from `run()` is used by the calling code which then wraps `”failed to create new cluster”`.